sphinx_needs/builder.py

```diff
@@ -160,3 +160,75 @@ def build_needumls_pumls(app: Sphinx, _exception: Exception) -> None:
         needs_builder.set_environment(env)
 
     needs_builder.finish()
+
+
+class NeedsIdBuilder(Builder):
+
```

Comment:
Does any empty line between function-definition and related docstring even work?